tools/mmp/driver.cs

```diff
@@ -354,6 +354,10 @@ static void Main2 (string [] args)
 					}
 				},
 				{ "link-prohibited-frameworks", "Natively link against prohibited (rejected by AppStore) frameworks", v => { LinkProhibitedFrameworks = true; } },
+				{ "warn-on-type-ref=", "Warn if any of the comma-separated types is referenced by assemblies - both before and after linking", v => {
```

Comment:
@spouliot it would be nice to document the format for the type string (`Namespace:Type`? `Namespace.Type`? Or just `Type`? What about nested types?)